tasks/base/CentOS-7/install_dependencies.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Ideally this should be a seperate PR since its unrelated, but its small enough to be okay this time :-)